ntbs-service/Migrations/20191115122928_AddBaseAlertClassWithExampleDerivedClass.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddBaseAlertClassWithExampleDerivedClass : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Alert",
+                columns: table => new
+                {
+                    AlertId = table.Column<int>(nullable: false)
+                        .Annotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn),
+                    NotificationId = table.Column<int>(nullable: true),
+                    CreationDate = table.Column<DateTime>(nullable: false),
+                    TbServiceCode = table.Column<string>(maxLength: 16, nullable: true),
+                    CaseManagerEmail = table.Column<string>(maxLength: 64, nullable: true),
+                    HospitalId = table.Column<Guid>(nullable: true),
+                    AlertType = table.Column<string>(maxLength: 30, nullable: false),
+                    AlertStatus = table.Column<string>(maxLength: 30, nullable: false),
+                    ClosureDate = table.Column<DateTime>(nullable: false),
+                    ClosingUserId = table.Column<string>(maxLength: 64, nullable: true),
+                    Discriminator = table.Column<string>(nullable: false),

```

Comment:
This discriminator is automatically added when using the TPH scheme and in this case is duplicating the AlertType's purpose - not sure if we want to delete the discriminator or try and use it instead of the alertType, more info can be found here https://docs.microsoft.com/en-us/ef/core/modeling/relational/inheritance